elixir/apps/web/lib/web/router.ex

```diff
@@ -84,6 +84,10 @@ defmodule Web.Router do
       ## Email
       live "/sign_in/providers/email/:provider_id", SignIn.Email
     end
+  end
```

Comment:
`:redirect_if_authenticated` was interrupting the sign-in flow when IdP redirected back to the portal, because that redirect did not have "as" param (it's stored in cookie) so it just switched to browser session and sent user to the sites page.

Reply:
Ah, that makes sense